src/test/ui/issue-51116.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0282]: type annotations needed
+  --> $DIR/issue-51116.rs:16:13
+   |
+LL |         for tile in row {
+   |             ---- consider giving this a type
```

Comment:
*Can* you give `tile` a type? I don't think you can. It seems like it'd be better to put the suggestion on `tiles`...